src/type-check.rkt

```diff
@@ -1,19 +1,26 @@
 #lang racket
 (require "common.rkt" "syntax/syntax.rkt" "errors.rkt")
-(provide assert-program-type! assert-expression-type!)
+(provide assert-program-type! assert-expression-type! type-of get-sigs argtypes->rtype)
 
-(define (get-sig fun-name num-args)
+(define (get-sigs fun-name num-args)
   (if (and (operator? fun-name) (hash-has-key? (operator-info fun-name 'type) num-args))
       (hash-ref (operator-info fun-name 'type) num-args)
       (if (hash-has-key? (operator-info fun-name 'type) '*)
           (hash-ref (operator-info fun-name 'type) '*)
           #f)))
 
-(define (get-params fun-name num-args)
-  (and (get-sig fun-name num-args) (car (get-sig fun-name num-args))))
+;; (define (get-params fun-name num-args)
+;;   (and (get-sigs fun-name num-args) (car (get-sigs fun-name num-args))))
 
-(define (get-rt-type fun-name num-args)
-  (and (get-sig fun-name num-args) (cadr (get-sig fun-name num-args))))
+;; (define (get-rt-type fun-name num-args)
+;;   (and (get-sigs fun-name num-args) (cadr (get-sigs fun-name num-args))))
```

Comment:
Instead of commenting these out, best just to delete them. The old versions are always available in version control.

Reply:
deleted